src/constants/path.js

```diff
@@ -0,0 +1,7 @@
+const path = {
+  home: '/',
+  cart: '/cart',
+  productDetail: '/product-detail'
+}
```

Comment:
đưa vào config và dùng kiểu config.route.[name].path

```suggestion
const path = {
  HOME: {
  path:'/',
  component: Home
  },
  CART: {
      path:'/cart',
      component: Cart
        },
...
}
```